apps/web/app/api/(internal)/pipeline/lib/handleIntegrations.ts

```
@@ -367,16 +367,21 @@ const buildNotionPayloadProperties = (
       case "select":
-        return {
-          name: value,
-        };
+        if (!value) return null;
+        if (typeof value === "string") {
+          // Replace commas
+          const sanitizedValue = value.replace(/,/g, "");
+          return {
+            name: sanitizedValue,
+          };
+        }
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)